### DISCUSSION TITLE
Give Scope doesn't match requested scope

### DISCUSSION THREAD

**Discussion by nicedevil007** [alexkn/node-red-contrib-homeconnect#54] [Q&A]
Heya,

I hope I configured everything as it should be but I get this here:

`{"error":"invalid_scope","error_description":"given scope doesn't match requested scope"}`

On the devpage I did something like this:
![image](https://user-images.githubusercontent.com/17103076/145709722-ae22d3f1-9815-454f-ad80-7b4574c2be5f.png)

And in node-red I did it like this:
![image](https://user-images.githubusercontent.com/17103076/145709765-cb365409-350d-4935-8f3c-be7ba0cde3b8.png)

And if both don't match I get this:
![image](https://user-images.githubusercontent.com/17103076/145709818-b77a4a65-cbe7-435e-9789-04f0afb31d88.png)
In english: `The user used is not assigned to this application in the Home Connect Developer Portal.`

**Comment by alexkn** [comment]
Your scopes are not correct
https://api-docs.home-connect.com/authorization#authorization-scopes
